src/pages/home/report/ReportActionsView.js

```diff
@@ -104,7 +103,19 @@ class ReportActionsView extends React.Component {
         AppState.addEventListener('change', this.onVisibilityChange);
         subscribeToReportTypingEvents(this.props.reportID);
         this.keyboardEvent = Keyboard.addListener('keyboardDidShow', this.scrollToListBottom);
-        this.recordMaxAction();
+        updateLastReadActionID(this.props.reportID);
+
+        // Since we want the New marker to remain in place even if newer messages come in, we set it once on mount.
+        // We determine the last read action by deducting the number of unread actions from the total number.
+        const lastReadSequenceNumber = this.props.report.unreadActionCount === 0
+            ? 0
+            : this.props.report.maxSequenceNumber - this.props.report.unreadActionCount;
+
+        // We are adding 1 to the last read sequence number because we want the New marker displayed over the
+        // first unread sequence. If there are no unread actions, there's no need to display it.
+        if (lastReadSequenceNumber > 0) {
+            setNewMarkerPosition(this.props.reportID, lastReadSequenceNumber + 1);
+        }
```

Comment:
>If there are no unread actions, there's no need to display it.

So should we set it to `0` instead of `0 + 1` ?

This change didn't resolve the issue I was seeing.

Reply:
You are completely right. Should be fixed now. I moved the +1 upwards so we only add it when != 0